fiftyone/plugins/secrets.py

```diff
@@ -101,20 +104,35 @@ def _get_secrets_client():
     return client()
 
 
-class SecretsDictionary:
+class SecretsDictionary(Mapping[str, str]):
```

Comment:
just checking: is this syntax valid in Python 3.7? We still promise support for it

Reply:
removed